src/store/staking/staking.slice.ts

```diff
@@ -149,7 +118,11 @@ export const stakingSlice = createSlice({
     },
 
     selectStake: (state, { payload }: PayloadAction<number>) => {
```

Comment:
So we're still identifying the stake with its date, but loading all the stake to the store, correct?

Reply:
Yep, that's right. We could also pass complete stake data in the payload of this action and just put it in the store, but that would require more changes.